src/crate/client/sqlalchemy/__init__.py

```diff
@@ -23,10 +23,24 @@
 from .dialect import CrateDialect
 from .sa_version import SA_1_4, SA_VERSION
 
-# SQLAlchemy 1.3 does not have the `exec_driver_sql` method.
+
 if SA_VERSION < SA_1_4:
+    import textwrap
+    import warnings
+
+    # SQLAlchemy 1.3 is effectively EOL.
+    SA13_DEPRECATION_WARNING = textwrap.dedent("""
+    WARNING: SQLAlchemy 1.3 is currently in maintenance status and will soon be EOL.
+    With SQLAlchemy 2 around the corner, this will be even more imminent.
+    Future versions of the CrateDB SQLAlchemy dialect will drop support for SQLAlchemy 1.3.
+    It is recommended that you transition to using SQLAlchemy 1.4 or 2.0.
+    """.lstrip("\n"))
```

Comment:
2.0 was released stable last night, and 1.3 is now officially EOL as per the table on https://www.sqlalchemy.org/download.html.

Reply:
Thanks. I've adjusted the admonition text correspondingly.